src/java/org/apache/cassandra/repair/RepairRunnable.java

```diff
@@ -74,44 +77,50 @@
 import org.apache.cassandra.tracing.TraceState;
 import org.apache.cassandra.tracing.Tracing;
 import org.apache.cassandra.transport.messages.ResultMessage;
-import org.apache.cassandra.utils.FBUtilities;
-import org.apache.cassandra.utils.Throwables;
-import org.apache.cassandra.utils.WrappedRunnable;
 import org.apache.cassandra.utils.progress.ProgressEvent;
 import org.apache.cassandra.utils.progress.ProgressEventNotifier;
 import org.apache.cassandra.utils.progress.ProgressEventType;
 import org.apache.cassandra.utils.progress.ProgressListener;
 
-import static org.apache.cassandra.concurrent.ExecutorFactory.Global.executorFactory;
 import static org.apache.cassandra.repair.state.AbstractState.COMPLETE;
 import static org.apache.cassandra.repair.state.AbstractState.INIT;
 import static org.apache.cassandra.service.QueryState.forInternalCalls;
-import static org.apache.cassandra.utils.Clock.Global.currentTimeMillis;
-import static org.apache.cassandra.utils.Clock.Global.nanoTime;
 
-public class RepairRunnable implements Runnable, ProgressEventNotifier, RepairNotifier
-{
+public class RepairRunnable implements Runnable, ProgressEventNotifier, RepairNotifier {
```

Comment:
What if we took the hint from some of your other naming...and just related `RepairRunnable` -> `RepairCoordinator`?

Reply:
are you saying `RepairRunner` and `RepairRunnable` are confusing?  😱